### ISSUE TITLE
Deprecation notice: "`formatError` is deprecated and replaced by `customFormatErrorFn`. It will be removed in version 1.0.0."

### ISSUE THREAD

**Issue by jlengstorf** [gatsbyjs/gatsby#16951]
On a fresh install of `gatsby@2.13.75`, the following error message started showing up today:

```
`formatError` is deprecated and replaced by `customFormatErrorFn`. It will be removed in version 1.0.0.
```

It prints every second or two and doesn't seem to stop:

<img width="642" alt="Screen Shot 2019-08-21 at 7 33 35 PM" src="https://user-images.githubusercontent.com/163561/63481543-9486d880-c44a-11e9-88cb-0d24086ae568.png">

I haven't dug into this at all, but it looks like something upstream?

**Comment by vihanb** [comment]
I fixed this by downgrading to 2.13.70 but from what I can tell the issue is caused here:

https://github.com/gatsbyjs/gatsby/blob/194a97c3854e3b8e4fcdd5b1eab1d72450dea867/packages/gatsby/src/commands/develop.js#L159

It looks like 9e165476a0557a63dd6cc7098008e651134511cb caused this which updated the express-graphql version to fix another issue but it caused this.

**Comment by kwangkim** [comment]
Or you may downgrade `express-graphql`  by adding
```
  "resolutions": {
    "express-graphql":"0.7.1"
  },
```
in your `package.json` file.

**Comment by wardpeet** [comment]
Thanks for opening this issue. We fixed it in `gatsby@2.13.76`
